e2e-tests/.ci/server.generate.sh

```diff
@@ -59,13 +58,10 @@ services:
       MM_PLUGINSETTINGS_ENABLED: "true"
       MM_PLUGINSETTINGS_ENABLEUPLOADS: "true"
       MM_PLUGINSETTINGS_AUTOMATICPREPACKAGEDPLUGINS: "true"
```

Comment:
These 3 should be modified during setup too and not as env variable for there are tests that modify their values.